.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Test
-        run: cargo test --manifest-path ${{ matrix.manifest }} ${{ matrix.additional-args }}
+        run: cargo test --manifest-path ${{ matrix.manifest }} ${{ matrix.additional-args }} --test *
```

Comment:
this will avoid test examples, as examples/kerberos.rs require a real KDC and server to work

Reply:
You probably meant `--tests` (with a final `s`), but this is NOT required. Examples an never _run_ as tests, they are only checked for compilation errors. You can read more here: https://doc.rust-lang.org/cargo/commands/cargo-test.html#target-selection
Note that even if we specified `--exemples`, it would only run _unit tests_ defined in example code, and that would be desirable.

Reply:
updated, the main problem was CI check complains about the feature used in the kerberos example, updated the the example, now it's guarded by cfg feature attribute.